source/adios2/engine/dataman/DataManReader.cpp

```diff
@@ -94,7 +94,19 @@ void DataManReader::Init()
             }
         }
         jmsg["stream_mode"] = "receiver";
-//        m_Man.add_stream(jmsg);
+        */
+
+        int n_Transports = 1;
+        std::vector<Params> para(n_Transports);
+
+        for (unsigned int i = 0; i < para.size(); i++)
+        {
+            para[i]["type"] = "wan";
+            para[i]["transport"] = "zmq";
+            para[i]["name"] = "stream";
+            para[i]["ipaddress"] = "127.0.0.1";
+        }
+        m_Man.OpenWANTransports("zmq", adios2::OpenMode::Read, para, true);
 
         std::string method_type;
```

Comment:
switch to camelCase: methodType  https://github.com/ornladios/ADIOS2/wiki/Variables-Scope,-Functions,-and-Namespaces